.evergreen/config.yml.in

```diff
@@ -87,7 +87,7 @@ functions:
       params:
         script: |
           ${PREPARE_SHELL}
-          git clone --depth 1 https://github.com/mongodb-labs/drivers-evergreen-tools.git $DRIVERS_TOOLS
+          git clone --depth 1 --branch DRIVERS-1983 https://github.com/mongodb-labs/drivers-evergreen-tools.git $DRIVERS_TOOLS
```

Comment:
This change must be reverted before merging - only here to make the tests pass since that branch getting merge is a requirement for this PR.

Reply:
Maybe we could merge this if we get it all ready to go in, and then when the tools PR is merged we can just do a oneliner to make this go back to pulling from master. As long as we expect it to go in soon. My thinking is we're waiting to hear on when all the teams want to contend with the failures that they'll all start seeing when this goes in. 

We might as well proceed with our fix if that makes sense? (@dariakp @baileympearson)

Reply:
That's not a terrible idea, but then we should file a follow up and schedule it immediately to make sure it doesn't get lost

Reply:
Drivers tools has been merged so this change has now been reverted.